README.md

```diff
@@ -23,7 +23,7 @@ jobs:
     steps:
       - name: <customize name>
         id: <customize id>
-        uses: CircleCI-Public/trigger-circleci-pipeline-action@v1.0
+        uses: CircleCI-Public/trigger-circleci-pipeline-action@v1.0.4
```

Comment:
When I copy pasted this I realised I wasn't on the latest version of the action. You may also want to maintain a `v1` tag and update that on every release.